src/aes_key.c

```diff
@@ -488,6 +489,9 @@ zpc_aes_key_import_clear(struct zpc_aes_key *aes_key, const unsigned char *key)
 		goto ret;
 	}
 
+	if (aes_key->type == ZPC_AES_KEY_TYPE_EP11)
```

Comment:
Who defines the type of the key? Shouldn't it in case of EP11 always be the one with header prepended? Would you ever create one with overlayed header?

Reply:
Yes, this is a libzpc constant that applies for aes ep11 keys in general (in contrast to CCA). EP11 type keys always get the header:
# define ZPC_AES_KEY_TYPE_CCA_DATA     1
# define ZPC_AES_KEY_TYPE_CCA_CIPHER   2
# define ZPC_AES_KEY_TYPE_EP11         3